Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -54,6 +54,7 @@ class BrowserViewController: UIViewController {
     var libraryDrawerViewController: DrawerViewController?
     var webViewContainer: UIView!
     var urlBar: URLBarView!
+    var urlBarHeight: Constraint!
```

Comment:
nit: could we change this variable name to `urlBarHeightConstraint`

Reply:
done.